js/util/pipeline-cubic.js

```diff
@@ -28,6 +28,7 @@ var RttEstimator = require('./rtt-estimator.js').RttEstimator;
 var Pipeline = require('./pipeline.js').Pipeline;
 var LOG = require('../log.js').Log.LOG;
 
+Log.LOG = 4;
```

Comment:
I think this was put here for debugging. Can you remove it? (The library should not set the LOG level. That is for the application to set.)

Reply:
Oops... I forgot to remove it.

Reply:
I pushed the new code.